packages/storybook/stories/components/HorizontalScroll.stories.tsx

```diff
@@ -88,33 +88,37 @@ export const HorizontalScrollSquadStory: Story = {
         memberPostingRole: SourceMemberRole.Admin,
         memberInviteRole: SourceMemberRole.Admin,
         image: 'https://via.placeholder.com/150',
-        handle: 'squad-handle'
+        handle: 'squad-handle',
       },
     };
 
     return (
-      <>
-        <HorizontalScroll title={title} className='gap-8 py-4'>
-          {new Array(30).fill(null).map((item, i) => (
-            /* eslint-disable react/no-array-index-key */
-            <UnfeaturedSquadGrid
-              {...args}
-              key={`horizontal scroll item ${i}`}
-            />
-          ))}
-        </HorizontalScroll>
-      </>
+      <QueryClientProvider>
+        <AuthContext.Provider value={{}}>
```

Comment:
Are these necessary, just asking since they weren't before.

Reply:
The storybook is not rendering without it.